src/main/java/uk/gov/hmcts/probate/controller/LifeEventController.java

```diff
@@ -30,4 +34,23 @@
         lifeEventService.verifyDeathRecord(caseDetails, securityUtils.getSecurityDTO());
         return ResponseEntity.ok(callbackResponseTransformer.updateTaskList(request));
     }
+
+    @PostMapping(path = "/updateWithSystemNumber")
+    public ResponseEntity<CallbackResponse> updateWithSystemNumber(@RequestBody CallbackRequest request) {
+        final Integer systemNumber = request.getCaseDetails().getData().getDeathRecordSystemNumber();
+        final DeathRecord deathRecord = lifeEventService.getDeathRecordById(systemNumber);
+        final CallbackResponse response = callbackResponseTransformer.updateTaskList(request);
+        response.getData().setDeathRecord(deathRecord);
+        return ResponseEntity.ok(response);
+    }
+
+    @PostMapping(path = "/updateWithSystemNumber2")
```

Comment:
can you give some more relevant name to this mapping . Also a brief comment on what it does would help may be

Reply:
I have now removed this endpoint and implemented it without a second callback